awesomplete.js

```diff
@@ -46,7 +46,7 @@ var _ = function (input, o) {
 	});
 
 	this.ul = $.create("ul", {
-		hidden: "",
+		hidden: "hidden",
```

Comment:
$.create when working with attributes would do:

```
element['hidden'] = '';
```

Which is for boolean attribute would be the same as:

```
element['hidden'] = false;
```

So completer would not be hidden by default.

I don't remember why I didn't used `hidden: false` here. Maybe there was some bug with it. I'll check this later.